app/toolbar_overrides/ems_cloud_center.rb

```diff
@@ -0,0 +1,33 @@
+module ManageIQ
+  module Providers
+    module Amazon
+      module ToolbarOverrides
+        class EmsCloudCenter < ::ApplicationHelper::Toolbar::Override
```

Comment:
move this to app/<foo>/manageiq/providers/amazon/toolbar_overrides/ems_cloud_center.rb

...where <foo> could be any of the autoload directories...I'm thinking maybe helpers or a new directory?  We can put whatever we want really...if it's a "new" directory, it just needs to be in application.rb in core, but that's easy.  What is the structure you use for toolbar stuff in ui-classic?

Reply:
sure, can move it from 'app/toolbar_overrides' to where you wrote. will give it a shot

in ui-classic the toolbars live under `app/helpers/application_helper/toolbar/` each toolbar has it's class

so in the provider repos I'd like to have `<something>/toolbar_overrides/` and then each overrride in it's own class